Cargo.toml

```diff
@@ -277,11 +277,11 @@ tar = "0.4.40"
 tempfile = "3.5.0"
 thiserror = "1.0.48"
 time = "0.3.30"
-tokio = "1.33.0"
-tokio-stream = "0.1.14"
-tokio-test = "0.4.2"
+tokio = "1.39.1"
+tokio-stream = "0.1.15"
+tokio-test = "0.4.4"
 tokio-tungstenite = { version = "0.20.1" }
-tokio-util = "0.7.10"
+tokio-util = "0.7.11"
```

Comment:
why not specify only minor, without patch versions?

Reply:
There is no good reason, just inertia. But it's a good point, I think I will start explicitly skip patch versions for >1.0 in the future